compiler/damlc/daml-doc/src/DA/Daml/Doc/HaddockParse.hs

```diff
@@ -532,9 +543,7 @@ tyConAnchor DocCtx{..} tycon = do
         mod = maybe dc_modname getModulename (nameModule_maybe ghcName)
         anchorFn
             | isClassTyCon tycon = classAnchor
-            | isDataTyCon tycon = dataAnchor
             | otherwise = typeAnchor
-    guard (not (isWiredInName ghcName))
```

Comment:
Why don't we need this guard anymore?

Reply:
I'm not sure it was ever necessary. The guard was there because at the time I wrote this function there wasn't a way to check that we only link to anchors that are actually available. But then I went ahead and added the RenderOut monoid that solves that problem, so it's fine to have more anchors than we produce in general.

Reply:
Thanks for the explanation.